### PR TITLE
[ANCHOR-603] Add fee_details documentation

### PR DESCRIPTION
**Pull request by Ifropc**

### CODE REVIEW

File: static/assets/rpc-methods/do_stellar_payment.json [hunk 1]
```diff
@@ -132,6 +132,35 @@
...
+            },
+            "asset": {
+              "type": "string"
+            },
+            "details": {
+              "type": "array",
+              "description": "Details about fee charged",
+              "items": {
+                "properties": {
+                  "name": {
```
Comment (philipliu) [inline static/assets/rpc-methods/do_stellar_payment.json:150]: Shouldn't `name` and `amount` be required?

File: static/assets/rpc-methods/notify_interactive_flow_completed.json [hunk 2]
```diff
@@ -91,6 +91,40 @@
         ]
       }
     },
+    {
+      "name": "fee_details",
+      "description": "Description of fees charged by the anchor",
```
Comment (philipliu) [inline static/assets/rpc-methods/notify_interactive_flow_completed.json:96]: Should we update the description so it's consistent with the ones in the other RPCs?

**Review by philipliu — COMMENTED** [review]
I think we would also need to update the integration guides. Do we plan to do it in another PR?

File: static/assets/rpc-methods/notify_interactive_flow_completed.json (hunk 2 above)
Comment (Ifropc) [reply static/assets/rpc-methods/notify_interactive_flow_completed.json:96]: Done

**Review by philipliu — APPROVED** [review]

**Pull request merged**